src/Common/src/TypeSystem/IL/Stubs/DynamicInvokeMethodThunk.cs

```diff
@@ -243,10 +243,12 @@ public override MethodIL EmitIL()
                 Context.GetWellKnownType(WellKnownType.Void);
 
             MethodSignature thisCallMethodSig = new MethodSignature(0, 0, returnType, targetMethodSignature);
-            thisCallSiteSetupStream.Emit(ILOpcode.calli, emitter.NewToken(thisCallMethodSig));
+            CalliIntrinsic.EmitTransformedCalli(emitter, thisCallSiteSetupStream, thisCallMethodSig);
+            //thisCallSiteSetupStream.Emit(ILOpcode.calli, emitter.NewToken(thisCallMethodSig));
```

Comment:
I kept it like this everywhere to make it easier for Sergey to test it out (no opportunity to mess up which codestream to emit in and waste time with that). No particular need though, just consistency with the rest of the places at this point.